src/containers/Form.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EntityDataModelApi, DataApi } from 'lattice';
+
+import FormView from '../components/FormView';
+
+class Form extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			reportInfo: {
+				1: '',
+				2: '',
+				3: null,
+				4: '',
+				5: '',
+				6: '',
+				7: '',
+				8: '',
+				9: null,
+				10: '',
+				11: ''
+			},
+			consumerInfo: {
+				'13a': '',
```

Comment:
When I remove the quotes from '13a', it throw this error: "Identifier directly after number". I could make them all have quotes for consistency.
It's a bit of a moot point, however, because I think these keys should reflect the same keys we'll use for the baltimorehealthreport entity type's property types, e.g. 'disposition'. Do you agree?